src/drivers/config.ts

```diff
@@ -41,17 +43,19 @@ export const saveConfig = async (orgId: string) => {
 
   const config = orgData.config ?? bootstrapConfig;
 
-  print2(`Saving config to ${CONFIG_FILE_PATH}.`);
+  const configFilePath = getConfigFilePath();
 
-  const dir = path.dirname(CONFIG_FILE_PATH);
+  print2(`Saving config to ${configFilePath}.`);
+
+  const dir = path.dirname(configFilePath);
   await fs.mkdir(dir, { recursive: true });
-  await fs.writeFile(CONFIG_FILE_PATH, JSON.stringify(config), { mode: "600" });
+  await fs.writeFile(configFilePath, JSON.stringify(config), { mode: "600" });
```

Comment:
cc: @bgberr, I'm a little concerned with how this interacts with proxycommand 🤔. might be worth syncing up tomorrow if ya'll have a minute during collab.